src/infrastructure/app.py

```diff
@@ -20,10 +19,11 @@
     invitations.router,
 )
 
-# Declare your middlewares here
-middlewares: Iterable[Type[BaseHTTPMiddleware]] = (
-    middlewares_.ErrorsHandlingMiddleware,
-    middlewares_.DatabaseTransactionMiddleware,
+# Declare your middlewares and their options here
+middlewares: Iterable[tuple[Type[middlewares_.Middleware], dict]] = (
+    (middlewares_.ErrorsHandlingMiddleware, {}),
+    (middlewares_.DatabaseTransactionMiddleware, {}),
```

Comment:
It seems order is not correct, CORS should be the first, Errors second and Database is the last

Reply:
True. Forgot to reorder that. Thanks!